.github/workflows/ci-nix.yml

```diff
@@ -422,23 +417,7 @@ jobs:
           # https://tech.aufomm.com/how-to-build-multi-arch-docker-image-on-nixos
           for container in $containers; do
             image="fedimint/$container"
-            if [ "$GITHUB_REF" = "refs/heads/master" ]; then
```

Comment:
Makes me think ... could we do this like ... daily or weekly maybe ? Just like what the plan is with the quick fuzzing?

Reply:
I think the difference with fuzzing is it's still valuable running infrequently since it may catch bugs. If we don't have any consumers of the `master` image then I think we can just drop them.

In my dev cycle, I've still had to do `nix build -L .#container.<bin>` if I need something more frequent than a release image, so the `master` images have never been useful on my end.

Let me know if you see it differently or you know of any projects downstream relying on our `master` image publishing.

Reply:
Makes sense. We can always bring this code back from git log if needed.